src/commandHandlers/bio.ts

```diff
@@ -59,33 +79,25 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
                     const url = `https://nominatim.openstreetmap.org/?addressdetails=1&q=${encodeURIComponent(data)}&format=json&limit=1`;
                     const response = await axios.default.get(url);
                     data = response.data ? response.data[0] : {};
+                    updateBio(data,field,message);
                 } catch (e) {
                     log.error(`ERROR: Couldn't get location ${data}`);
                 }
                 break;
             case 'twitter':
-                data = `https://twitter.com/${data}`
+                if(data.match(/^@?(\w){1,15}$/i)) {
```

Comment:
Could you either make a comment or store this Regex on a function with a specific name? It would help with readbility since we could see by the function name what is being done here.
From the issue associated with the PR, maybe we could call it `isValidTwitterUsername`.

What do you think?

Reply:
Sounds good ill make it into a function

Reply:
Personally I think \w is a weird regex that is not very readable. IMO this should be:
```
data.replace('@', '').match(/^[a-z0-9_]{1,15}$/i)
```

That regex is actually readable as to what it does in my opinion.